requirements.txt

```diff
@@ -210,3 +210,4 @@ anyjson==0.3.3
 pylama==6.4.0
 # sha256: yY7zECUyGAxtDrno1pd0u97Y2QtnGYjuRV1W_h9xGos
 django-guardian==1.3
+-e git://github.com/jotes/django-pg-fts.git@django_1.8.4#egg=django-pg-fts
```

Comment:
This should be something like (take a look at other github links):
# sha256: GtaJi7PO-64H-TBAHFrCTgvQqaD7N3RzLCXyyD1nrnA
https://github.com/jotes/django-pg-fts/archive/django_1.8.4.zip#egg=django-pg-fts==1.0

No -e, https protocol, and version is also needed as part of the egg.